### ISSUE TITLE
provideFeedback boolean must be a string

### ISSUE THREAD

**Issue by damienmarchandfr** [twilio/twilio-node#272]
**Version: 3.0.0**

### Code Snippet
```javascript
var twilio = require('twilio');
var client = new twilio(config.campaign.twilio.sid, config.campaign.twilio.token);
var smsBody = {};
smsBody.messagingServiceSid = 'messagingServiceSid';
smsBody.from = '+33644601238';
smsBody.body = 'toto';
smsBody.to = '+33759403492';
smsBody.id = sms.id;
smsBody.statusCallback = 'https://********.com/cb/twilio/update';
smsBody.provideFeedback =  false;

client.messages.create(smsBody)
```

### Exception / Log
```
_http_outgoing.js:456
    throw new TypeError('First argument must be a string or Buffer');
    ^
```

### Steps to Reproduce
1. provideFeedback is a Boolean and not a String

### Feature Request
Boolean must be convert to String. It's confusing to send 'false' instead of false

**Comment by efossier** [comment]
@damienmarchandfr Hey, we have a fix for this https://github.com/twilio/twilio-node/pull/262 unfortunately haven't had the time to release the library yet. In the meantime you can work around this by passing in `provideFeedback` as the string `"false"`.

**Comment by efossier** [comment]
@damienmarchandfr This has been released today in our latest versions `3.5.0` and `3.5.0-alpha-1`. Please let us know if the issue persists!
